File: ios/CustomMethods.m
//
//  CustomMethods.m
//  iosApp
//
//  Created by Abhijit Utkarsh on 30/05/24.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridge.h>

@interface RCT_EXTERN_MODULE(CustomMethods, NSObject)

RCT_EXTERN_METHOD(MyMethod: (NSString *) params)

@end

File: ios/CustomMethods.swift
//
//  CustomMethods.swift
//  iosApp
//
//  Created by Abhijit Utkarsh on 30/05/24.
//

import Foundation

@objc(CustomMethods) class CustomMethods: NSObject {
  @objc static func requireMainQueueSetup() ->Bool { return true}
  
  @objc public func MyMethod(_ param:String){
    print(param)
  }
}
